lightningd/plugin.c

```diff
@@ -1277,9 +1277,12 @@ bool plugins_send_getmanifest(struct plugins *plugins)
 
 	/* Spawn the plugin processes before entering the io_loop */
 	list_for_each_safe(&plugins->plugins, p, next, list) {
+		const char *err;
+
 		if (p->plugin_state != UNCONFIGURED)
 			continue;
-		if (plugin_send_getmanifest(p)) {
+		err = plugin_send_getmanifest(p);
+		if (!err) {
```

Comment:
Could this leak `err`?

Reply:
Almost: it's allocated off of the plugin which we'd be freeing if there is an error.

Reply:
Yeah, I use this pattern  through this code quite a bit.